### PR TITLE
Ability to style picker container.

### PR DESCRIPTION
**Pull request by dbonner1987**
Added new styling to the view containing the iOS picker to allow for styling the background/border behind and around the picker. Helps contain the component instead of the picker feeling entirely separate to that of the touch handler.

### CODE REVIEW

**Comment by dbonner1987** [comment]
I thought an example of this functionality would be useful.

20160923_tcomb-form-native_picker-example

**Comment by alvaromb** [comment]
Will work on this tomorrow, sorry for the delay!

**Comment by dbonner1987** [comment]
No problem, currently using my fork in an app for the time being.

File: lib/stylesheets/bootstrap.js [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 */
 'use strict';
+import {Platform} from 'react-native';
```
Comment (alvaromb) [inline lib/stylesheets/bootstrap.js]: Please leave an empty line between `use strict` and the `import. And also add spaces between `Platform`. Just to keep in line with the rest of the code style.

File: lib/templates/bootstrap/select.ios.js [hunk 2]
```diff
@@ -43,8 +46,8 @@ class CollapsiblePickerIOS extends React.Component {
 
     const height = (this.state.isCollapsed) ? 0 : UIPICKER_HEIGHT;
     return (
-      <View>
-        <TouchableOpacity style={touchableStyle}
+      <View style={[pickerContainer, (!this.state.isCollapsed) ? pickerContainerOpen : {}]}>
```
Comment (alvaromb) [inline lib/templates/bootstrap/select.ios.js]: I'm curious, why both `pickerContainer` and `pickerContainerOpen` should be applied as the style of the `View` and why not one or another?

I'm not saying it's not ok, I just want to know the use-case of this style merge.

Comment (dbonner1987) [inline lib/templates/bootstrap/select.ios.js]: Utilising both removes the need for duplicate code and allows the styling to work in a manner similar to CSS.

For instance, if all you wanted to do was change the `borderColor` for the for open state, if you were to use one or the other, the style declaration would like this:

```javascript
pickerContainer: {
  normal: {
    marginBottom: 4,
    borderRadius: 4,
    borderColor: BORDER_COLOR,
    borderWidth: 1,
  },
  error: {
    marginBottom: 4,
    borderRadius: 4,
    borderColor: ERROR_COLOR,
    borderWidth: 1,
  },
  open: {
    marginBottom: 4,
    borderRadius: 4,
    borderColor: 'blue',
    borderWidth: 1,
  }
}
```

However, if you use multiple "state" declarations, you can reduce the code to the following:

```javascript
pickerContainer: {
  normal: {
    marginBottom: 4,
    borderRadius: 4,
    borderColor: BORDER_COLOR,
    borderWidth: 1,
  },
  error: {
    borderColor: ERROR_COLOR,
  },
  open: {
    borderColor: OPEN_BORDER_COLOR,
  }
}
```

It's a personal preference and makes the stylesheets smaller and manageable.

**Comment by javiercr** [comment]
As this PR was approved, will it be finally merged into master? 👍

**Comment by gcanti** [comment]
> As this PR was approved, will it be finally merged into master?

Oh, sorry @dbonner1987, I didn't notice. Thanks @javiercr for pointing out

**Comment by gcanti** [comment]
Released in https://github.com/gcanti/tcomb-form-native/releases/tag/v0.6.3

**Comment by javiercr** [comment]
Thanks! :D

**Pull request merged**